Not finding lib.rs.cc on macos when running cmake build

Thanks for your project!
I get the following issue when running ```cmake --build .``` on macos:

```
clang: error: no such file or directory: '/Users/{USER}/cpp-with-rust/target/cxxbridge/blobstore/src/lib.rs.cc'
clang: error: no input files
make[2]: *** [CMakeFiles/cpp_with_rust.dir/target/cxxbridge/blobstore/src/lib.rs.cc.o] Error 1
make[1]: *** [CMakeFiles/cpp_with_rust.dir/all] Error 2
make: *** [all] Error 2
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -29,6 +29,11 @@ target_include_directories(
 
 target_link_libraries(cpp_with_rust ${BLOBSTORE_LIB})
 
+set_target_properties(
+    cpp_with_rust
+    PROPERTIES ADDITIONAL_CLEAN_FILES ${CARGO_TARGET_DIR}
+)
+
 # Windows-only configuration
 if(WIN32)
     target_link_libraries(cpp_with_rust userenv ws2_32 bcrypt)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,6 +16,7 @@ add_custom_command(
         COMMAND cargo build --manifest-path ${CARGO_MANIFEST}
         DEPENDS ${BLOBSTORE_SOURCE_FILE}
         USES_TERMINAL
+        COMMENT "Running cargo..."
 )
 
 add_executable(cpp_with_rust src/main.cpp src/multibuf.cpp ${BLOBSTORE_BRIDGE_CPP})
@@ -30,8 +31,8 @@ target_include_directories(
 target_link_libraries(cpp_with_rust ${BLOBSTORE_LIB})
 
 set_target_properties(
-    cpp_with_rust
-    PROPERTIES ADDITIONAL_CLEAN_FILES ${CARGO_TARGET_DIR}
+        cpp_with_rust
+        PROPERTIES ADDITIONAL_CLEAN_FILES ${CARGO_TARGET_DIR}
 )
 
 # Windows-only configuration

